[Docs][Clang] Missing DR status for C++23-era papers in cxx_status.html

frederick-vs-ja opened the pull request:
List the following C++23-era WG21 papers as Defect Reports in cxx_status.html as per WG21 meeting minutes.

 - [P1949R7](https://wg21.link/p1949r7) (C++ Identifier Syntax using Unicode Standard Annex 31)
 - [P2156R1](https://wg21.link/p2156r1) (Allow Duplicate Attributes)
 - [P2036R3](https://wg21.link/p2036r3) (Change scope of lambda _trailing-return-type_)
 - [P2468R2](https://wg21.link/p2468r2) (The Equality Operator You Are Looking For)
 - [P2327R1](https://wg21.link/p2327r1) (De-deprecating `volatile` compound operations)
 - [P2493R0](https://wg21.link/p2493r0) (Missing feature test macros for C++20 core papers)
 - [P2513R3](https://wg21.link/p2513r3) (`char8_t` Compatibility and Portability Fix)
 - [P2460R2](https://wg21.link/p2460r2) (Relax requirements on `wchar_t` to match existing practices)
 - [P2579R0](https://wg21.link/p2579r0) (Mitigation strategies for [P2036](https://wg21.link/p2036) ”Changing scope for lambda _trailing-return-type_”)

cor3ntin reviewed (approved):
Thanks!

Endilll commented:
Thank you for updating our status page!

The following papers were applied as defect reports. Have you considered them?
N4891: P2156 (Allow Duplicate Attributes) 
N4916: P2280 (Using unknown pointers and references in constant expressions)

I agree with your analysis in "Unchanged" section. As for [P2460R2 Relax requirements on `wchar_t` to match existing practices](https://wg21.link/p2460r2), it indeed seem like a pure wording change, but I'm not sure as well. Maybe @cor3ntin has an opinion on this.

frederick-vs-ja commented:
@Endilll Thanks for notification! P2280 was already handled, while P2156 was previously missed.
@cor3ntin I've add the DR status to P2156R1 and force-pushed.

Endilll commented:
@frederick-vs-ja Just to be clear, are you waiting for anyone to merge this?

frederick-vs-ja commented:
@Endilll Oh, I'm sorry for late replying. I wasn't wating for anybody and think this can be merged.

Endilll commented:
@frederick-vs-ja PR description is going to be used as a commit message. You can edit it. Let me know when it's ready.

frederick-vs-ja committed:
[Docs][Clang] DR status for C++23-era papers in cxx_status.html

List the following C++23-era WG21 papers as Defect Reports in cxx_status.html as per WG21 meeting minutes.

- P1949R7 (C++ Identifier Syntax using Unicode Standard Annex 31)
- P2156R1 (Allow Duplicate Attributes)
- P2036R3 (Change scope of lambda trailing-return-type)
- P2468R2 (The Equality Operator You Are Looking For)
- P2327R1 (De-deprecating volatile compound operations)
- P2493R0 (Missing feature test macros for C++20 core papers)
- P2513R3 (char8_t Compatibility and Portability Fix)
- P2460R2 (Relax requirements on wchar_t to match existing practices)
- P2579R0 (Mitigation strategies for P2036 ”Changing scope for lambda trailing-return-type”)

frederick-vs-ja commented:
@Endilll I think it's ready now. Sorry for late replying again.

Endilll commented:
All 3 buildbot failures are unrelated.
